### ISSUE TITLE
[Feature Request] Acrylic glass/blur effect as a chat background

### ISSUE THREAD

**Issue by kniazevgeny** [telegramdesktop/tdesktop#16626]
### Is your feature request related to a problem?
No

### Describe the solution you'd like
I'd like to see in some future updates acrylic glass/blur effect. In windows 11 insider preview it can be found in windows terminal 
![image](https://user-images.githubusercontent.com/41857481/125855117-20b2afc9-84f5-4d9e-81f3-adffcadf8a27.png)
and in some other apps. This improvement would match win11 style

**Comment by Grackins** [comment]
Hi!
Do you have more details in mind?
For example do you mean just the messages background should be somehow transparent and the chats and folders list should keep the same style as before?

**Comment by kniazevgeny** [comment]
I'm talking about folders 
![sample1](https://user-images.githubusercontent.com/41857481/125933341-d876acb0-40bc-483d-9b9c-06cc85d38fe5.png)
and chat background 
![sample2](https://user-images.githubusercontent.com/41857481/125933366-dd950290-19d8-4cf9-b8de-47aca376f4e4.png)

**Comment by marcselman** [comment]
I agree. When Telegram Desktop first came out it was fantastic with smooth animations and a great UI.
It's still very nice, but with Windows 11 now and the move toward the Fluent design language Telegram Desktop feels a little out of place. Supporting Acrylic and Mica surfaces would be a good step. Also rounded corners don't work on Telegram Desktop in Windows 11 which also feels off.

**Comment by hessam-kk** [comment]
Would you please reopen this issue?

**Comment by hessam-kk** [comment]
what new thoughts?? don't you mind to add acrylic option for telegram?? even windows terminal has this option...

**Comment by hessam-kk** [comment]
what new thoughts?? don't you mind to add acrylic option for telegram?? even windows terminal has this option...

**Comment by PeacedoseDev** [comment]
I'm looking for the update on that issue. At least just allow transparency, acrylic and mica can be handled by Windhawk or MicaForEveryone

**Comment by StepanAsk** [comment]
MicaForEveryone is the future

**Comment by martymarty004** [comment]
On Linux, KDE Plasma already handles transparent backgrounds with blur/translucency effects, and it works with other UI elements set with alpha 0.

I tried using a transparent PNG as background, but it renders as white, and it doesn't seem to be supported as of #29773.

**Comment by user123456000** [comment]
@martymarty004 Have you found any way to make the background transparent?

**Comment by pratikpatel8982** [comment]
@martymarty004 @user123456000 I've created a patch to prevent transparent images rendering as white #30698
